src/explore-education-statistics-admin/src/pages/methodology/components/MethodologySummaryForm.tsx

```diff
@@ -19,7 +19,8 @@ const errorMappings = [
   mapFieldErrors<MethodologySummaryFormValues>({
     target: 'title',
     messages: {
-      SlugNotUnique: 'Choose a unique title',
+      SlugNotUnique: 'Used by other methodology. Choose a unique title',
```

Comment:
Have these been run my Cam?  Not sure "Used by methodology redirect" would mean much to an Analyst, although it is definitely more useful than just a plain "Choose a unique title" I guess :)

Reply:
I haven't - I'll do that.

My main thought in including the "Used by methodology redirect" error was mainly for our benefit if we're debugging an issue.